cfgmgr/intfmgr.cpp

```diff
@@ -426,7 +426,12 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
 
         if (is_lo)
         {
-            addLoopbackIntf(alias);
+            if (m_loopbackIntfList.find(alias) == m_loopbackIntfList.end())
```

Comment:
so `nat_zone ` handling is done only `if not is_lo`. Where is it applying nat_zone on loopback interface?

Reply:
Not pushing the nat_zone value to APPL_DB when it got applied for loopback interface. nat_zone for loopback interface gets handled in natmgr code for setting iptables mangle tables rules (https://github.com/Azure/sonic-swss/blob/master/cfgmgr/natmgr.cpp#L901)